packages/offix-client/src/OfflineClient.ts

```diff
@@ -12,8 +12,9 @@ import {
 } from "offix-offline";
 import { ApolloOfflineClient } from "./ApolloOfflineClient";
 import { MutationHelperOptions, createMutationOptions } from "offix-cache";
-import { FetchResult } from "apollo-link";
+import { ApolloLink,FetchResult } from "apollo-link";
```

Comment:
no need to import `ApolloLink`

Reply:
true, that was part of the initial plan to resolve the issue we had, thanks!